app/decorators/program_decorator.rb

```diff
@@ -0,0 +1,26 @@
+class ProgramDecorator < Draper::Decorator
+  delegate_all
+
+  def to_diffable_resource
```

Comment:
Assignment Branch Condition size for to_diffable_resource is too high. [31.14/15]